pkg/sanity/controller.go

```diff
@@ -416,7 +416,8 @@ var _ = DescribeSanity("Controller Service [Controller Server]", func(sc *TestCo
 			Expect(serverError.Code()).To(Equal(codes.InvalidArgument))
 		})
 
-		It("should return appropriate values SingleNodeWriter NoCapacity Type:Mount", func() {
+		// TODO: whether CreateVolume request with no capacity should fail or not depends on driver implementation
```

Comment:
The commit message mentions added new tests, but all I find in the diff are changes to existing tests?

In this case here, the test doesn't even get changed, just its name.

I'm confused, sorry.

Reply:
The idea of this PR is to make possible running all the above-mentioned test cases for raw volumes.
Currently when sanity tests are executed with the flag `--csi.testvolumeaccesstype=block` above mentioned test cases are still executed for `mount` volumes.
My patch is based on the point, that all those cases use `MakeCreateVolumeReq` function for volume creation and I've just made it to produce raw volume when tests are executed for raw volumes.

Reply:
@pohly In commit message I wanted to say that raw volumes support was added to all those tests.
I've checked that list of tests cases and sure that when tests are executed with `--csi.testvolumeaccesstype=block` flag, all calls which bear volume access type are constructed with proper volume access type.